### PR TITLE
Add Phillip as a maintainer

### PR DESCRIPTION
**Pull request by tswast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

/CC @cpcloud

### CODE REVIEW

**Comment by tswast** [comment]
@conda-forge-admin, please rerender

**Comment by tswast** [comment]
@conda-forge-admin please rerender

**Comment by tswast** [comment]
@conda-forge/core Having trouble rerendering this one.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1001
+  number: 1002
```
Comment (scopatz) [inline recipe/meta.yaml]: You don't need to bump the build number just to add maintainers

File: recipe/meta.yaml [hunk 2]
```diff
@@ -45,4 +45,5 @@ extra:
     - parthea
     - jreback
     - tswast
-    - jonparrott
```
Comment (scopatz) [inline recipe/meta.yaml]: @jonparrott - are you OK with being removed?

Comment (tswast) [reply recipe/meta.yaml]: That account is no longer active. It belongs to a teammate of mine.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -45,4 +45,5 @@ extra:
...
+    - theacodes
+    - cpcloud
```
Comment (scopatz) [inline recipe/meta.yaml]: @theacodes @cpcloud - are you OK with being added as maintainers?

Comment (cpcloud) [reply recipe/meta.yaml]: Yep.

Comment (tswast) [reply recipe/meta.yaml]: I just realized Thea is OOO until May. We can send a separate PR to add her once she's back.

**Pull request merged**